src/main/java/com/mongodb/jdbc/MongoPreparedStatement.java

```diff
@@ -29,10 +29,10 @@
     private MongoStatement statement;
 
     public MongoPreparedStatement(
-            String sql, MongoConnection conn, String databaseName, boolean relaxed)
+            String sql, MongoStatement statement)
```

Comment:
This change we agreed to leave out of the last PR, so I did it now. `prepareStatement` is part of the `Connection` class, so it makes sense to do now.